packages/application-shell/src/components/custom-view-shell-authenticated/custom-view-shell-authenticated.tsx

```diff
@@ -95,9 +98,15 @@ function CustomViewShellAuthenticated(
                               projectDataLocale={props.dataLocale}
                               environment={props.environment}
                             >
-                              <CustomViewWithPermissionCheck>
-                                {props.children}
-                              </CustomViewWithPermissionCheck>
+                              <CustomViewFlopFlipProvider
```

Comment:
I don't mind but the SetupFlipFlip is also camelcase on the Flip. Just to consistent on the casing here and the filename.

Reply:
Sorry but I don't understand.

The current name of the component is already camel casing `FlopFlip`.
Could you maybe give an example of how the name should looks like?
Thanks.

Reply:
Sorry for not making the point clear. I double checked myself which I couldn't yesterday.

I just wanted to make sure that `setup-flop-flip-provider.tsx` and this file follow the same `flop-flip` casing. The library itself isn't cased like it. Hence my "stumble" on the review.